routers/repo/setting.go

```diff
@@ -727,3 +730,53 @@ func init() {
 		panic(err)
 	}
 }
+
+// UpdateAvatarSetting update repo's avatar
+// FIXME: limit size.
```

Comment:
You must limit size before we merge - just stick a config in it doesn't need a great UI.

Reply:
Fixed